README.md

```diff
@@ -1,22 +1,158 @@
 
+[![Website](https://img.shields.io/badge/%40-Virtomize-%231e828c)](https://virtomize.com)
+[![Blog](https://img.shields.io/badge/%40-Blog-%23379099)](https://blog.virtomize.com)
+[![Docs](https://img.shields.io/badge/%40-Docs-%2369acb2)](https://docu.virtomize.com)
+[![Buy](https://img.shields.io/badge/Buy-License-%2382babf)](https://virtomize.com/pricing)
+[![License](https://img.shields.io/badge/License-EULA-%2382babf.svg)](LICENSE)
 
-[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=VBXHBYFU44T5W&source=url)
-[![Website](https://img.shields.io/badge/website-virtomize-%23379099)](https://virtomize.com)
-[![Documentation](https://img.shields.io/badge/docu-reference-%231e828b)](https://docu.virtomize.com)
-[![Blog](https://img.shields.io/badge/virtomize-blog-%231e828b)](https://blog.virtomize.com)
-[![Twitter](https://img.shields.io/badge/-twitter-grey?logo=twitter)](https://twitter.com/virtomize)
-[![Xing](https://img.shields.io/badge/-xing-grey?logo=xing)](https://www.xing.com/companies/virtomizegmbh)
-[![Linked-In](https://img.shields.io/badge/-linkedin-grey?logo=linkedin)](https://www.linkedin.com/company/virtomize/)
-[![Facebook](https://img.shields.io/badge/-facebook-grey?logo=facebook)](https://www.facebook.com/groups/virtomize/)
+[![Twitter URL](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2Fvirtomize)](https://twitter.com/virtomize)
+[![Xing](https://img.shields.io/badge/xing-%20-blue.svg?style=social&logo=xing)](https://www.xing.com/companies/virtomizegmbh)
+[![LinkedIn](https://img.shields.io/badge/linkedIn-%20-blue.svg?style=social&logo=linkedin)](https://www.linkedin.com/company/virtomize/)
 
-# Virtomize
+![Logo](https://virtomize.com/logo-text.svg)
 
-![Logo](https://github.com/Virtomize/virtomize/blob/master/logo.svg) 
 
-## Create - Deploy - Provision
+# Create - Deploy - Provision
 
-Virtomize is a software automation solution on top of your hypervisor.
+[Virtomize](https://virtomize.com) is a commercial software build for hypervisors to create, install and configure virtual machines with only a few clicks or via [REST API](https://docu.virtomize.com).
```

Comment:
Virtomize is a commercial software for simplifying interaction with a hypervisor, e.g. to create, install and configure virtual machines with only a few clicks or via REST API.

Reply:
Hm how do you like the new one?